CODE/Index.html

```diff
@@ -59,7 +69,13 @@ <h2>By Carl Zimmer
         Only a few species stand out in these studies, and by comparing them, scientists have identified some shared
         factors. The animals have big brains relative to their body size, they live for a long time, and they can form
         long-lasting social bonds.
-    </p>
+    </p4>
+
+    <figure>
+        <img src="images/maxresdefault.jpg" alt=" Octopus on the sea bed">
+        <figcaption>An image of an octopus via youtube. https://www.youtube.com/watch?v=rX-YiYHahoo. </figcaption>
```

Comment:
If you want to display a link, you should either make it a link like so:
```suggestion
        <figcaption>An image of an octopus via <a href="https://www.youtube.com/watch?v=rX-YiYHahoo">YouTube</a>.</figcaption>
```
or:
```suggestion
        <figcaption>An image of an octopus via YouTube. <a href="https://www.youtube.com/watch?v=rX-YiYHahoo">https://www.youtube.com/watch?v=rX-YiYHahoo</a>.</figcaption>
```